### PR TITLE
Increase 1010d test coverage

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

The purpose of this PR is to increase the unit test coverage for ivc-champva form 10-10d.

- Added several unit tests for the front-end of form 10-10d

## Testing done

- Wrote several unit tests
- Ran `yarn test:coverage-app ivc-champva/10-10D` to verify new and existing tests worked

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Testing  |before | after|

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

**Review by balexandr — DISMISSED** [review]
lgtm

File: src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+});
+
+describe('Custom relationship pattern', () => {
+  it('should allow past-tense wording', async () => {
+    const relationship = relationshipToVeteranUI({ tense: 'past' });
+    expect(relationship.relationshipToVeteran?.['ui:title']).to.contain(
+      'What’s your relationship',
+    );
+  });
+});
```
Comment (micahchiang) [inline src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js:29]: Can this all be refactored into 1 `desrcibe` block? Or, is there a benefit I'm not seeing to  keeping these in 3 separate blocks?

File: src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js
Comment (micahchiang) [inline src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js:1]: do the callbacks in each of these `it` statements need to be `async`? I don't see `await` being used anywhere.

File: src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js (hunk 1 above)
Comment (michaelclement) [reply src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js]: No reason for them to be separate, just didn't consider the redundancy when putting these in. [Fixed in this commit here](https://github.com/department-of-veterans-affairs/vets-website/commit/542f46c2363558913b2d39a038ef6403f745f46a)

File: src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js
Comment (michaelclement) [reply src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js:1]: [Fixed in this commit](https://github.com/department-of-veterans-affairs/vets-website/commit/542f46c2363558913b2d39a038ef6403f745f46a) - they don't need to be async, I got a little copy/paste happy on these I think

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**